pkg/configuration/configuration.go

```diff
@@ -66,6 +66,10 @@ const (
 	// DefaultTestingMode specifies whether the services should run in testing mode.
 	DefaultTestingMode = false
 
+	varAuthClientLibraryURL = "auth_client.library_url"
+	// DefaultAuthClientLibraryURL is the default auth library location.
+	DefaultAuthClientLibraryURL = "https://keycloak.service/auth/js/keycloak.js"
```

Comment:
I don't understand why you use such a default value. It's not valid, is it?

Reply:
(same question for the `DefaultAuthClientPublicKeysURL` below)

Reply:
Yeah, there is no valid url for that and the default will not work. This is sort of a mandatory config. The key url was updated in the other PR to be default "", so that by default no keys are loaded.